src/makeCalculator.js

```
@@ -37,7 +37,39 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    result: 0,
+
+    operate(toDo, number) {
+      this.result = toDo(this.result, number);
+
+      return this;
+    },
+
+    add(accum, number) {
+      return accum + number;
+    },
+
+    subtract(accum, number) {
+      return accum - number;
+    },
+
+    multiply(accum, number) {
+      return accum * number;
+    },
+
+    divide(accum, number) {
+      return number > 0 ? accum / number : false;
```

it's better to throw an error if value is `0` not just `false`